<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetClass</class>
 <widget class="QWidget" name="WidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse vel nunc id mi rutrum molestie. Nulla augue nisi, suscipit sed odio dapibus, suscipit varius arcu. Aliquam vel sollicitudin ipsum. Vestibulum ornare, augue eu mollis mollis, risus ligula tempor nisi, ac ultricies justo nisi ac nibh. Curabitur a venenatis velit.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="textButton">
     <property name="text">
      <string>Text Colour</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bgroundButton">
     <property name="text">
      <string>Background Colour</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="fontButton">
     <property name="text">
      <string>Font</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Widget.qrc"/>
 </resources>
 <connections/>
</ui>
